### PR TITLE
Feature/update images sf rebase and styles

### PR DESCRIPTION
**Pull request by hectoruch**
I update styles for header.
That is not like the our app styles because we need change the DOM and also others styles.
But is the best I can do with the DOM exist right now on public pages.
@nerik @clementprdhomme @andresgnlez @tiagojsag

### CODE REVIEW

**Comment by andresgnlez** [comment]
ready to merge for me.

**Comment by tiagojsag** [comment]
All pages need to use the two new images - please remove all old background images and make sure all pages have one of the two new.

**Comment by andresgnlez** [comment]
Please, use references "the two new images" doesn't help at all :/ No idea what you are talking about.

File: public/form/css/styles.css [hunk 1]
```diff
@@ -705,23 +747,11 @@ hr {
...
 
 .bg.log {
-  background-image: url('../img/sky.png');
-}
-
-@media only screen and (min-width: 768px) {
-  .bg.log {
-    background-image: url('../img/sky@2x.png');
-  }
+  background-image: url('../img/login.jpg');
```
Comment (tiagojsag) [inline public/form/css/styles.css]: @hectoruch please confirm with @clementprdhomme that this is the way to go

**Comment by tiagojsag** [comment]
See that all comments from https://github.com/Vizzuality/GlobalFishingWatch/pull/197 are addressed

**Comment by tiagojsag** [comment]
Please make sure ALL pages have attribution and that the attribution matches the image

**Comment by andresgnlez** [comment]
is this OK for merge, @tiagojsag ?

**Comment by tiagojsag** [comment]
menu seems to be broken on Iphone 6s plus landscape - sometimes it shows up, sometimes it doesnt, sometimes has background, sometimes not... can you confirm it's not just on my phone?

**Pull request merged**